src/ScenarioPreview/ScenarioPreview.tsx

```diff
@@ -0,0 +1,152 @@
+import { Box, Divider, Link, Stack, Typography } from "@mui/material";
+import { LabelChipGroup, LabelChipGroupProps } from "../LabelSelector";
+
+import DateLabel from "../DateLabel/DateLabel";
+import FileLabel from "../FileLabel/FileLabel";
+import FormatLabel from "../FormatLabel/FormatLabel";
+import FormatVersionLabel from "../FormatVersionLabel/FormatVersionLabel";
+import NoWrapTypography from "../NoWrapTypography/NoWrapTypography";
+import React from "react";
+import RoadLabel from "../RoadLabel/RoadLabel";
+import { ScenarioPreviewProps } from "./ScenarioPreview.types";
+import UserLabel from "../UserLabel/UserLabel";
+
+export function ScenarioPreview({
+  name,
+  href,
+  image,
+  description,
+  format,
+  formatVersion,
+  file,
+  createdAt,
+  user,
+  label = [],
+  roadLabel,
+  sx
+}: ScenarioPreviewProps) {
+  function hasOptionalProperty() {
+    return label?.length > 0 || createdAt || user;
+  }
+
+  const labelChips: LabelChipGroupProps["chips"] = label?.map(l => ({
+    clickable: false,
+    color: l.color,
+    label: l.name,
+    size: "small"
+  }));
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      gap={1}
+      minWidth={0}
+      fontFamily="Montserrat"
+      data-testid="scenario-preview-wrapper"
+      sx={{ ...sx }}
+    >
+      <Box>
+        <Stack direction="row" spacing={1} minWidth={0}>
+          <img
+            src={image}
+            alt="scenario-image"
+            style={{
+              height: "44px",
+              marginBottom: "auto",
+              marginTop: "auto",
+              objectFit: "contain",
+              width: "78px"
+            }}
+          />
+          <Stack direction="column" minWidth={0}>
+            <NoWrapTypography>
+              <Link
+                href={href}
+                target="_blank"
+                color="primary"
+                variant="subtitle2"
+                underline="hover"
+                data-testid="scenario-preview-name"
+              >
+                {name}
+              </Link>
+            </NoWrapTypography>
+
+            <Typography
+              color="textPrimary"
+              data-testid="scenario-preview-description"
+              sx={{
+                WebkitBoxOrient: "vertical",
+                WebkitLineClamp: 2,
+                display: "-webkit-box",
+                overflow: "hidden",
+                textOverflow: "ellipsis"
```

Comment:
I think when we use the webkit props, these are irrelevant and do nothing, so I suggest these are removed.

Reply:
Seems like Firefox deals with it differently, than chrome. 
![image](https://github.com/user-attachments/assets/7b422df1-da0c-474d-ad41-1fffe5c7a4bd)

In this case I suggest we just leave it as is, so it works on both browsers